tests/test_generic.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from pyro.generic import handlers, infer, pyro, pyro_backend
-from pyro.generic.testing import MODELS
```

Comment:
These tests are also covered in `pyro-api` and can be removed from here.

Reply:
I'd like to keep these tests in both repos so as to prevent both breakage of api and breakage of implementation. Does that make sense?

I think we can eventually move those tests into the more automatic `pyroapi.tests`, so at least code won't be duplicated.

Reply:
Sure, that makes sense!